core/config/toml/types.go

```diff
@@ -642,11 +649,43 @@ func (w *WebServer) setFrom(f *WebServer) {
 		w.HTTPMaxSize = v
 	}
 
+	w.LDAP.setFrom(&f.LDAP)
 	w.MFA.setFrom(&f.MFA)
 	w.RateLimit.setFrom(&f.RateLimit)
 	w.TLS.setFrom(&f.TLS)
 }
 
+func (w *WebServer) ValidateConfig() (err error) {
+	// Validate WebServer config, assert LDAP fields when AuthMethod set to LDAP
+	if *w.AuthenticationMethod == string(sessions.LDAPAuth) {
+		if *w.LDAP.BaseDN == "" {
+			err = multierr.Append(err, configutils.ErrInvalid{Name: "LDAP.BaseDN", Value: "", Msg: "LDAP BaseDN can not be empty"})
```

Comment:
You can omit `Value` on these:
```suggestion
			err = multierr.Append(err, configutils.ErrInvalid{Name: "LDAP.BaseDN", Msg: "LDAP BaseDN can not be empty"})
```
Or there is an `ErrEmpty`

Reply:
Done 🙂